core/src/com/unciv/logic/battle/Battle.kt

```diff
@@ -429,7 +429,7 @@ object Battle {
             interceptor.attacksThisTurn++
 
             val attackerName = attacker.getName()
-            val interceptorName = interceptor.name
+            val interceptorName = interceptor.displayName
```

Comment:
The interceptor name should be relevant only to the civ who the interceptor belongs to